homeassistant/components/shelly/switch.py

```diff
@@ -66,6 +66,11 @@ class RpcSwitchDescription(RpcEntityDescription, SwitchEntityDescription):
     sub_key="value",
 )
 
+RPC_SCRIPT_SWITCH = RpcSwitchDescription(
+    key="script",
+    sub_key="running",
```

Comment:
If I remember correctly we agreed that these entities should be disabled by default.
```suggestion
    sub_key="running",
    entity_registry_enabled_default=False,
```